### PR TITLE
Add release notes for 0.154

### PR DESCRIPTION
**Pull request by cberner**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.154.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+=============
+Release 0.154
+=============
+
+General Changes
+---------------
+
+* Add ``code-cache-collection-threshold`` config which controls when Presto
+  will attempt to force collection of the JVM code cache.
+* Add support for ``CREATE TABLE LIKE``.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.154.rst]:
```
* Add support for using ``LIKE`` with :doc:`/sql/create-table`.

**Comment by electrum** [comment]
```
* Add ``DESCRIBE INPUT`` for prepared statements.
* Reduce default code cache collection threshold to ``40%``.

**Comment by electrum** [comment]
```
Hive Changes
------------

* Fix handling of metastore cache TTL. With the introduction of the
  transaction per-transaction cache, the cache timeout was reset
  after each access, which means cache entries might never expire.

**Comment by haozhun** [comment]
```
* Fix bug that may cause "Input symbols do not match output symbols" when writing to bucketed tables.
```

**Comment by electrum** [comment]
```
* Fix potential *"Input symbols do not match output symbols"* error when writing to bucketed tables.

**Comment by martint** [comment]
```
* Fix planning issue that could cause ``JOIN`` queries involving functions 
  that return null on non-null input to produce incorrect results.
* Fix regression that would cause certain queries involving uncorrelated 
  subqueries in ``IN`` predicates to fail during planning.
* Add support for ``DESCRIBE INPUT`` to describe the requirements for 
  the input parameters to a prepared statement.
* Improve performance of window functions with identical partitioning and 
  ordering but different frame specifications.
```

**Comment by electrum** [comment]
Merged, thanks!

**Pull request closed without merging**